build.sbt

```diff
@@ -42,6 +42,7 @@ lazy val microservice = (project in file("."))
       "-Wconf:src=routes/.*:s",
       "-Wconf:src=html/.*:s",
     ),
+    Test / scalacOptions := scalacOptions.value,
     libraryDependencies ++= AppDependencies(),
     retrieveManaged := true,
     resolvers ++= Seq(Resolver.jcenterRepo),
```

Comment:
This was an additional change to remove the warnings when running it/tests:
[warn] Flag -deprecation set repeatedly
[warn] Flag -unchecked set repeatedly
[warn] Flag -encoding set repeatedly